bencyn/Questioner-React
# Setup UI and Redux

#### What does this PR do?
This PR setups the UI and redux state management
### Tasks to be completed?
- [x] - setup react with web-pack and babel
- [x] - setup project initial UI
- [x] - setup redux folder structure

### How can this be manually tested?

- Clone the repo and check out the `setup-ui`  branch

- setup npm
```
$ npm install
```

- run application
```
$ npm run dev
```
